README.md

```diff
@@ -1,10 +1,17 @@
 # Lamoda metric responder
 
+[![Build Status](https://travis-ci.org/lamoda/php-metrics.svg?branch=master)](https://travis-ci.org/lamoda/php-metrics)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/lamoda/php-metrics/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/lamoda/php-metrics/?branch=master)
+[![Code Coverage](https://scrutinizer-ci.com/g/lamoda/php-metrics/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/lamoda/php-metrics/?branch=master)
+[![Build Status](https://scrutinizer-ci.com/g/lamoda/php-metrics/badges/build.png?b=master)](https://scrutinizer-ci.com/g/lamoda/php-metrics/build-status/master)
+[![Latest Stable Version](https://poser.pugx.org/lamoda/metrics/v/stable)](https://packagist.org/packages/lamoda/metrics)
+[![SymfonyInsight](https://insight.symfony.com/projects/36e6e1fc-6e84-4766-b496-a100ea1dbdd6/mini.svg)](https://insight.symfony.com/projects/36e6e1fc-6e84-4766-b496-a100ea1dbdd6)
```

Comment:
This link leads nowhere (404) for other users. @pavelsavelyev will try to auth sensiolab for org to allow public inspections

Reply:
Temporarly delete link